neonvm/apis/neonvm/v1/virtualmachine_webhook.go

```diff
@@ -145,6 +145,11 @@ func (r *VirtualMachine) ValidateUpdate(old runtime.Object) (admission.Warnings,
 		}
 	}
 
+	// allow to change CPU scaling mode only if it's not set
+	if r.Spec.CpuScalingMode != nil && before.Spec.CpuScalingMode != nil && *r.Spec.CpuScalingMode != *before.Spec.CpuScalingMode {
+		return nil, fmt.Errorf(".spec.cpuScalingMode is immutable")
```

Comment:
suggestion: not quite "immutable" per se, but can only be set once, right? If so, should this message be different?

Reply:
changed message